spec/granite/table/table_spec.cr

```diff
@@ -1,7 +1,7 @@
 require "../../spec_helper"
 
 describe Granite::Table do
-  describe "#table_name" do
+  describe ".table_name" do
```

Comment:
This `#` syntax is common in many languages, and I think is preferred. It's the syntax the crystal docs use.

Reply:
I was going off of https://crystal-lang.org/reference/guides/testing.html#anatomy-of-a-spec-file:

```
For unit tests, it is recommended to follow the conventions for method names: 
Outer describe is the name of the class, inner describe targets methods. 
Instance methods are prefixed with `#`, class methods with `.`.
```

Reply:
Bah, I'm moving too fast today. Sorry. Class methods should start with a `.` yes.